README.md

```diff
@@ -137,18 +200,50 @@ Replaces a named policy with one from the provided file. This is
 usually a destructive invocation. Result is a dictionary object
```

Comment:
Replaces a named policy with a policy from the provided file.

Reply:
what do you mean? I think it is self explanation when we say destructive